docs/source/extend_kedro/plugins.md

```diff
@@ -44,6 +44,32 @@ Once the plugin is installed, you can run it as follows:
 kedro to_json
 ```
 
+## Extending Starters Template
+You can extend `kedro starter list` and `kedro new` by introducing your own `cookiecutter` template.
+
+You need to extend the starters by providing a list of  `KedroStarterSpec`, in this example it is defined in a file called `plugin.py`.
+
+```python
+starters = [
+    KedroStarterSpec(
+        alias="test_plugin_starter",
+        template_path="https://github.com/kedro-org/kedro-starters/",
```

Comment:
Does this work? We have `git+https://github.com/kedro-org/kedro-starters.git` in the source code?

Edit: just tried it and it does!

Reply:
Maybe we should be consistent about how we do it though. Probably URL to a git repo is a bit nicer than the `git+` address.